Packages/cdms2/Lib/dataset.py

```diff
@@ -245,14 +245,14 @@ def openDataset(uri,mode='r',template=None,dods=1,dpath=None, hostObj=None):
                 file = CdmsFile(uri,mode)
                 return file
             except:
-                raise "Error in DODS open of: ",uri
+                raise CDMSError, "Error in DODS open of: ",uri
         else:
             try:
                 datanode = loadURI(uri)
                 return datanode
             except:
                 datanode = loadURI(uri)
-                raise "Error in loadURI of: ",uri
+                raise CDMSError, "Error in loadURI of: ",uri
```

Comment:
I'd say let's make it clean so it will work when/if we move to Python 3.0. Since you brought that up, would you mind fixing it across the file? Thanks! Because the way it is now is also compatible with 2.x, so why bother changing it in the first place?